samples/client/petstore/spring-cloud-spring-pageable/src/main/java/org/openapitools/api/PetApi.java

```diff
@@ -165,7 +205,7 @@ ResponseEntity<List<Pet>> findPetsByStatus(
         produces = "application/json"
     )
     ResponseEntity<List<Pet>> findPetsByTags(
-        @NotNull @ApiParam(value = "Tags to filter by", required = true) @Valid @RequestParam(value = "tags", required = true) List<String> tags,
+        @NotNull @ApiParam(value = "Tags to filter by", required = true) @Valid @RequestParam(value = "tags", required = true) List&lt;String&gt; tags,
```

Comment:
Looks like something happened here which didn't replace with `<` and `>` properly

Reply:
Looks like a generation issue. I've just regenerate them and they are ok.